### ISSUE TITLE
Cannot update address_kana (and more) on JP custom account

### ISSUE THREAD

**Issue by floriangaerber** [stripe/stripe-php#366]
| Component | Version |
| --- | --- |
| PHP | 7.0.10 |
| `stripe/stripe-php` | 4.13.0 |
| Stripe API | 2017-06-05 (latest) (test mode) |

I am trying to update a JP custom accounts `legal_entity.address_kana.postal_code` property.

The request is submitted without error, but also without the `legal_entity.address_kana.postal_code` property.

I've used `Symfony\VarDumper` to inspect the `Stripe\Account` object and found the following:
- `$account->legal_entity` is an instance of `Stripe\AttachedObject`
- `$account->legal_entity->address` of a DE custom account is an instance of `Stripe\AttachedObject`
  - After updating a value, the corresponding field is listed in `unsavedValues` property of the `AttachedObject`
  - After using `$account->save();`, `unsavedValues` is empty again
  - Updating a DE custom account's address works (address parameters are submitted)
- `$account->legal_entity->address_kana` of a JP custom account is an instance of `Stripe\StripeObject`
  - After updating a value, the corresponding field is listed in `unsavedValues` property of the `StripeObject`
  - After using `$account->save();`, `unsavedValues` still lists the changed value, when no other fields were updated.
    - When also updating another value, in the same call, such as `first_name`, `address_kana` is reset to its previous state.
  - Updating a JP custom account does work (i.e. `legal_entity.first_name`, ...), but updating the address does not. (Neither kana nor kanji)

I had a brief look into it and believe the Stripe client fails to collect `address_kana` and `address_kanji` in `Stripe\StripeObject::serializeParameters()` [here](https://github.com/stripe/stripe-php/blob/master/lib/StripeObject.php#L235), because they aren't listed in `Stripe\StripeObject::nestedUpdatableAttributes` [here](https://github.com/stripe/stripe-php/blob/master/lib/StripeObject.php#L29).

### Appendix
<details>

**Minimal Example:**
```php
$account = Stripe\Account::retrieve($id);
$account->legal_entity->address_kana->postal_code = '1500001';
$account->save();
```

Result: No API call is made (checked logs), likely because client did not detect any changes.

**Verification:**

```php
$account = Stripe\Account::retrieve($id);
$account->legal_entity->address_kana->postal_code = '1500001';
$account->legal_entity->first_name = 'Max';
$account->save();
```
Result: API call is made successfully, logs show "Request POST body":
```json
{
  "legal_entity": {
    "first_name": "Max"
  }
}
```
</detail>

**Comment by remi-stripe** [comment]
Hey @floriangaerber, thanks a lot for the detailed bug report here. I came up to the same conclusion and was able to confirm it works once I changed the list of `nestedUpdatableAttributes`. I pushed a fix in https://github.com/stripe/stripe-php/pull/367 so hopefully we can merge it over the next few days.

**Comment by floriangaerber** [comment]
Thanks for the quick response! I just tried that locally and was about to report that as well.

**Comment by brandur-stripe** [comment]
Thanks for the report!

Fix released in 5.1.3.
